### PR TITLE
Bug Fix :1677

### PR DESCRIPTION
**Pull request by gagandeepp**
GitHub Issue (If applicable): #

https://github.com/unoplatform/uno/issues/1677

https://github.com/unoplatform/uno/issues/1677

## PR Type

What kind of change does this PR introduce?

## What is the current behavior?
Linked url is incorrect

When user click on bug template in contributing.md it redirects to 404 page

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ We want to hear about your experience, scenarios, and requirements.
 
 ### Report a bug
 
-If you think you've found a bug, please log a new issue in the [Uno Platform GitHub issue tracker](https://github.com/nventive/Uno/issues). When filing issues, please use our [bug filing template](.github/ISSUE_TEMPLATE.md).
+If you think you've found a bug, please log a new issue in the [Uno Platform GitHub issue tracker](https://github.com/unoplatform/uno/blob/master/.github/ISSUE_TEMPLATE/bug-report.md). When filing issues, please use our [bug filing template](.github/ISSUE_TEMPLATE.md).
```
Comment (jeromelaban) [inline CONTRIBUTING.md]:
```suggestion
If you think you've found a bug, please log a new issue in the [Uno Platform GitHub issue tracker](https://github.com/unoplatform/uno/issues/new/choose).
```

**Comment by gagandeepp** [comment]
@jeromelaban  please merge this as well

**Comment by ghuntley** [comment]
Thanks for your contribution, some feedback if you don't mind? Please don't do 👇 

> @jeromelaban please merge this as well

These two blogs posts on contributing code to open source projects are a very good read and explain the social etiquette: [Open Source Contribution Etiquette by Miguel de Icaza](https://tirania.org/blog/archive/2010/Dec-31.html) and [Don’t “Push” Your Pull Requests by Ilya Grigorik](https://www.igvita.com/2011/12/19/dont-push-your-pull-requests/).

Anyway, can you please drop by https://platform.uno/uno-is-joining-hacktoberfest-2019/ and follow the instructions on that page and we'll send you some swag.

**Comment by ghuntley** [comment]
@all-contributors please add @gagandeepp for docs

**Pull request merged**